src/Tizen.NUI/src/public/BaseComponents/TextFieldEvent.cs

```diff
@@ -73,6 +78,32 @@ public partial class TextField
             }
         }
 
+        /// <summary>
+        /// The CursorMoved event.
+        /// </summary>
+        /// This will be public opened after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public event EventHandler<CursorMovedEventArgs> CursorMoved
```

Comment:
How about renamed to `CursorPositionChanged`? because TextEditor provide `CursorPosition` and it inform changes of `CursorPosition`
```suggestion
        public event EventHandler<CursorMovedEventArgs> CursorPositionChanged
```